streamer/src/recvmmsg.rs

```diff
@@ -99,21 +99,22 @@ pub fn recv_mmsg(sock: &UdpSocket, packets: &mut [Packet]) -> io::Result<(usize,
         return Err(io::Error::last_os_error());
     }
     let mut npkts = 0;
+    let mut total_size = 0;
     addrs
         .iter()
         .zip(hdrs)
+        .zip(packets.iter_mut())
```

Comment:
I think we can use `izip` here, similar to above this line
https://github.com/solana-labs/solana/blob/0e4ede46d/streamer/src/recvmmsg.rs#L81

Reply:
Good call - done